sdk/servicebus/service-bus/test/internal/unit/streamingReceiver.spec.ts

```diff
@@ -124,31 +118,23 @@ describe("StreamingReceiver unit tests", () => {
     });
   });
 
-  describe("forever loop", () => {
-    let streamingReceiver: StreamingReceiver | undefined;
-
-    afterEach(() => {
-      return streamingReceiver?.close();
-    });
-  });
-
   it("onDetach calls through to init", async () => {
     const streamingReceiver = createTestStreamingReceiver("fakeEntityPath");
 
-    const subscribeImplMock = sinon.mock();
+    const subscribeImplMock = vi.fn();
 
     streamingReceiver["_subscribeImpl"] = subscribeImplMock;
     await streamingReceiver.onDetached(new Error("let's detach"));
-    assert.isTrue(subscribeImplMock.calledOnce);
+    assert.equal(subscribeImplMock.mock.calls.length, 1);
```

Comment:
Could we use `expect(closeLinkSpy).toHaveBeenCalledOnce()`?

Reply:
updated.